src/main/resources/alma/common/describedBy.xml

```diff
@@ -82,9 +82,14 @@
 			</entity>
 		</entity>
 		<entity name="sourceOrganization">
-			<data name="id" source="MNG  .g">
```

Comment:
Are you sure that those `MNG` fields contain the information we are looking for here?

Reply:
changed to "MNG  .a"

Reply:
Resolved in https://github.com/hbz/lobid-resources/pull/1100/commits/3c3109845839d9c49051c7f48bac0202cf0bd5ad#diff-946dcf62c20237a36122ec3ee9ef389c4c4b06f9e8c061e53e87543d42640236R89.